src/glewlwyd.h

```diff
@@ -32,17 +32,28 @@
 #include <jansson.h>
 #include <jwt.h>
 
-#define _GNU_SOURCE
-#define __USE_GNU
+#ifndef _GNU_SOURCE
+ #define _GNU_SOURCE
+#endif
+
+#ifndef __USE_GNU
+ #define __USE_GNU
+#endif
+
```

Comment:
You can create a new branch from the master branch, then in this new branch, put this modification, then create a pull request from the new branch.